### PR TITLE
Add FAQ

### PR DESCRIPTION
**Pull request by tarmolov**

### CODE REVIEW

**Comment by Flackus** [comment]
//cc @sipayRT @j0tunn

File: README.md [hunk 1]
```diff
@@ -33,3 +33,17 @@ plugins: {
     }
 }
 ```
+
+## FAQ
+### Tunnel to example.com closed. Exit code: 255
+Commonly it's a problem with permissions to the host (`example.com` in this example). You can mannually create an ssh tunnel. Let's assume that you want to open a tunnel from `localhost:8080` to `example.com:9000`.
```
Comment (Flackus) [inline README.md]: `Commonly` -> `Usually`?
`mannually` -> `manually`

File: README.md [hunk 2]
```diff
@@ -33,3 +33,17 @@ plugins: {
...
+
+Run the following command.
+```
+$ ssh example.com -R 9000:localhost:8080 -N
```
Comment (Flackus) [inline README.md]: Also, `-v` could be useful.

`-v      Verbose mode.  Causes ssh to print debugging messages about its progress.  This is helpful in debugging connection,
             authentication, and configuration problems.  Multiple -v options increase the verbosity.  The maximum is 3.`

**Comment by Flackus** [comment]
Thank you :)

**Comment by tarmolov** [comment]
Fixed

**Comment by sipayRT** [comment]
👍

**Comment by Flackus** [comment]
@tarmolov thanks again :)

**Pull request merged**